src/tests/class_tests/openms/data/AccurateMassSearchEngine_output1_consensusXML.mzTab

```diff
@@ -14,41 +24,41 @@ MTD	study_variable[8]-description	Accurate mass search result file.
+SMH	identifier	chemical_formula	smiles	inchi_key	description	exp_mass_to_charge	calc_mass_to_charge	charge	retention_time	taxid	species	database	database_version	spectra_ref	search_engine	best_search_engine_score[1]	search_engine_score[1]_ms_run[1]	search_engine_score[1]_ms_run[2]	search_engine_score[1]_ms_run[3]	search_engine_score[1]_ms_run[4]	search_engine_score[1]_ms_run[5]	search_engine_score[1]_ms_run[6]	search_engine_score[1]_ms_run[7]	search_engine_score[1]_ms_run[8]	search_engine_score[1]_ms_run[9]	search_engine_score[1]_ms_run[10]	modifications	smallmolecule_abundance_study_variable[1]	smallmolecule_abundance_stdev_study_variable[1]	smallmolecule_abundance_std_error_study_variable[1]	smallmolecule_abundance_study_variable[2]	smallmolecule_abundance_stdev_study_variable[2]	smallmolecule_abundance_std_error_study_variable[2]	smallmolecule_abundance_study_variable[3]	smallmolecule_abundance_stdev_study_variable[3]	smallmolecule_abundance_std_error_study_variable[3]	smallmolecule_abundance_study_variable[4]	smallmolecule_abundance_stdev_study_variable[4]	smallmolecule_abundance_std_error_study_variable[4]	smallmolecule_abundance_study_variable[5]	smallmolecule_abundance_stdev_study_variable[5]	smallmolecule_abundance_std_error_study_variable[5]	smallmolecule_abundance_study_variable[6]	smallmolecule_abundance_stdev_study_variable[6]	smallmolecule_abundance_std_error_study_variable[6]	smallmolecule_abundance_study_variable[7]	smallmolecule_abundance_stdev_study_variable[7]	smallmolecule_abundance_std_error_study_variable[7]	smallmolecule_abundance_study_variable[8]	smallmolecule_abundance_stdev_study_variable[8]	smallmolecule_abundance_std_error_study_variable[8]	smallmolecule_abundance_study_variable[9]	smallmolecule_abundance_stdev_study_variable[9]	smallmolecule_abundance_std_error_study_variable[9]	smallmolecule_abundance_study_variable[10]	smallmolecule_abundance_stdev_study_variable[10]	smallmolecule_abundance_std_error_study_variable[10]	opt_global_mz_ppm_error	opt_global_adduct_ion	opt_global_isosim_score	opt_global_MTint	opt_global_neutral_mass	opt_global_id_group
+SML	HMDB:HMDB14907	C27H41NO6	[H][C@@]12CC[C@](O)(C(=O)COC(=O)CN(CC)CC)[C@@]1(C)C[C@H](O)[C@@]1([H])[C@@]2([H])CCC2=CC(=O)CC[C@]12C	InChI=1S/C27H41NO6/c1-5-28(6-2)15-23(32)34-16-22(31)27(33)12-10-20-19-8-7-17-13-18(29)9-11-25(17,3)24(19)21(30)14-26(20,27)4/h13,19-21,24,30,33H,5-12,14-16H2,1-4H3/t19-,20-,21-,24+,25-,26-,27-/m0/s1	Hydrocortamate	539.31071622816296	539.309157845695722	1	921.389662323789025	null	null	HMDB	3.5	null	[, , AccurateMassSearch, ]	null	null	null	null	null	null	null	null	null	null	null	null	3.12282e06	0.0	0.0	6.58385e06	0.0	0.0	4.31776e06	0.0	0.0	8316.330078125	0.0	0.0	4.89429e06	0.0	0.0	4.96792e06	0.0	0.0	9657.2998046875	0.0	0.0	6282.509765625	0.0	0.0	0.0	0.0	0.0	2.5240599609375e04	0.0	0.0	2.889590218461454	M+CH3CN+Na;1+	-1	null	475.294946431467281	1
```

Comment:
Hmm search_engine_score_ms_run is always null? Doesnt AMS have a score? E.g. massdiff?

Reply:
Yes, currently "null" was used all the time, but I think using the mass error is a good idea. 
I guess, it is enough to use the mass error in ppm?

Reply:
Yeah I guess ppm. Renaming makes sense, too, if no effort.